config/deploy.rb

```diff
@@ -20,10 +23,4 @@
 # TASKS
```

Comment:
Looks great, same as the other PR, let's just remove the `# TASKS` section now that it's empty.